crates/bevy_internal/src/lib.rs

```diff
@@ -193,3 +190,40 @@ pub mod dynamic_plugin {
     //! Dynamic linking of plugins
     pub use bevy_dynamic_plugin::*;
 }
+
+/// `use bevy::prelude::*;` to import common components, bundles, and plugins.
+pub mod prelude {
+    #[doc(hidden)]
+    pub use crate::{
+        app::prelude::*, asset::prelude::*, core::prelude::*, core_pipeline::prelude::*,
+        ecs::prelude::*, hierarchy::prelude::*, input::prelude::*, log::prelude::*,
+        math::prelude::*, render::prelude::*, text::prelude::*, time::prelude::*,
+        transform::prelude::*, ui::prelude::*, utils::prelude::*, window::prelude::*,
+        DefaultPlugins, MinimalPlugins,
+    };
+
+    pub use bevy_derive::{bevy_main, Deref, DerefMut};
+}
+
+#[doc(hidden)]
```

Comment:
I don't think this should be hidden in docs.

Reply:
The reason I wrote that was because the original prelude had the doc hidden.

Reply:
What should I write instead since the CI requires documentation?

Reply:
> The reason I wrote that was because the original prelude had the doc hidden.

It's meant to hide re-exports, not the whole mod. 

> What should I write instead since the CI requires documentation?

Maybe something like: 
```rust
/// `use bevy::prelude_2d::*;` to import common components, bundles and plugins for creating 2d games.
```